src/content/data-plumbing.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "What your plumber can teach you about Data Pipelines"
+description: "What principles of flow, can we apply to tech, from our water management colleagues."
+created_at: 2023-02-19
```

Comment:
**nitpick:** you may want to update the `created_at` & `updated_at`dates

Reply:
Thanks Charlotte. I thought about that but as that's when I actually wrote and didn't change anything, I left them the same